### PR TITLE
feat(appsvc): Add compose for appsvc image

### PR DESCRIPTION
**Pull request by smulvih2**

### CODE REVIEW

File: docker-compose.appsvc.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    build:
+      context: docker/images/appsvc
+      args:
+        - BASE_IMAGE=${BASE_IMAGE}
+        - http_proxy=${HTTP_PROXY:-}
+        - https_proxy=${HTTP_PROXY:-}
+        - HTTP_PROXY=${HTTP_PROXY:-}
+        - HTTPS_PROXY=${HTTP_PROXY:-}
+        - NO_PROXY=hub,${NO_PROXY:-}
+        - no_proxy=hub,${NO_PR
```
Comment (sylus) [inline docker-compose.appsvc.yml]: Think missing some stuff here on the last line

Comment (smulvih2) [reply docker-compose.appsvc.yml]: @sylus wasn't sure if I needed to include this, but added here and will test shortly today.

File: docker-compose.appsvc.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
```
Comment (sylus) [inline docker-compose.appsvc.yml]:
```suggestion
        - no_proxy=hub,${NO_PROXY:-}
```

**Comment by sylus** [comment]
Let me know if works and I can merge! :)

**Comment by smulvih2** [comment]
Created a PR for stite-wxt so we don't forget to add a symlink to appsvc.yml file - [https://github.com/drupalwxt/site-wxt/pull/36](https://github.com/drupalwxt/site-wxt/pull/36)

**Pull request merged**